models/topics-model.js

```diff
@@ -48,3 +48,26 @@ exports.fetchAllArticles = async () => {
   );
   return rows;
 };
+
+exports.fetchCommentsByArticleId = async (id) => {
+  const { rows } = await db.query(
+    `
+      SELECT articles.article_id, comments.author, comments.body, comment_id, comments.created_at, comments.votes FROM articles LEFT JOIN comments
+      ON articles.article_id = comments.article_id
+      WHERE articles.article_id = $1
+      ORDER BY created_at DESC
+     `,
+    [id]
+  );
+  if (!rows[0])
+    return Promise.reject({
+      status: 404,
+      msg: `No article found for article_id ${id}`,
+    });
+  else if (rows[0].comment_id === null)
+    return Promise.reject({
+      status: 404,
+      msg: `No comments found for article_id ${id}`,
+    });
```

Comment:
This shouldn't be an error. Should just return an empty array. :)

Reply:
Was wondering about this one, that makes it easier to code anyway